README.md

```diff
@@ -92,6 +94,23 @@ else fails, try removing the virtual environment and reinstalling:
 [local5000]:http://localhost:5000/
 
 
+### Troubleshooting Possible Errors
+
+Should you get series of type errors that looks something like `npm ERR! typeerror Error: Missing required argument #1`, after running `npm install`, this is most likely due to running an older version of node. 
```

Comment:
Required versions should be stated under a new "Pre-requisites heading" rather than in troubleshooting.